src/luacov/runner.lua

```diff
@@ -397,6 +402,21 @@ local function has_hook_per_thread()
 end
 
 --------------------------------------------------
+-- Wraps a function to be used in a coroutine created via C API.
```

Comment:
On the contrary, it is always needed unless only LuaJIT is used. I'll add some notes.